eclipse-collections/src/main/java/org/eclipse/collections/impl/utility/Iterate.java

```diff
@@ -3118,7 +3163,11 @@ public static boolean contains(Iterable<?> iterable, Object value)
     }
 
     /**
+     * Also see {@link Iterate#groupByEach(Iterable, Function, MutableMultimap)}, {@link Iterate#toMultimap(Iterable, Function, Function, MutableMultimap)}.
```

Comment:
This one Javadoc comment using "Also see" and Javadoc links seems inconsistent with the rest.